.github/workflows/ci.yml

```diff
@@ -469,7 +469,7 @@ jobs:
       - run: python -m pytest -n 2 modin/pandas/test/test_general.py
       - run: python -m pytest modin/pandas/test/test_io.py
       - run: python -m pytest modin/experimental/pandas/test/test_io_exp.py
-      - run: pip install dfsql && pytest modin/experimental/sql/test/test_sql.py
+      - run: pip install dfsql "mindsdb_sql<=0.0.12" && pytest modin/experimental/sql/test/test_sql.py
```

Comment:
maybe we should move `dfsql` and `mindsdb_sql<=0.0.12` to the testing dependencies? they're located at [environment-dev.yml](https://github.com/modin-project/modin/blob/master/environment-dev.yml)

Reply:
Possible yes, it will be more easy to check deps versions in the one place such `environment-dev.yml/requirements-dev.txt`.

Reply:
`dfsql` and `mindsdb_sql` moved to `environment-dev.yml/requirements-dev.txt`

Reply:
There is an issue in moving `dfsql`/`mindsdb_sql` to test deps - these packages is provided only for linux. I propose to use solution from 0b8d805e6714384ceb3cf4857a81a020a0f624bc. @dchigarev, what do you think?

Reply:
okay, let's use 0b8d805e6714384ceb3cf4857a81a020a0f624bc solution